app/jobs/cache_num_published_solutions_on_exercise_job.rb

```diff
@@ -0,0 +1,9 @@
+class CacheNumPublishedSolutionsOnExerciseJob < ApplicationJob
+  def perform(exercise)
```

Comment:
Maybe be explicit about the queue being used? We do that in other jobs too.

```suggestion
  queue_as :default

  def perform(exercise)
```

Reply:
Good shout. Thanks.